release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -77,6 +77,29 @@ to customers based on data from the Red Hat Service Reliability Engineering
 (SRE) team, you might not immediately see notification in the web console that
 updates from version 4.2.z to 4.3 are available at initial release.
 
+[id="ocp-4-3-authentication"]
+=== Authentication
```

Comment:
I would consider making this a "Security" heading (I looked back and saw we had that heading in 4.1). And then maybe you could consider moving the item under "Certificates" below into this section as well. wdyt?

Reply:
I'm glad you brought that up. I was on the fence about what section to categorize this under :sweat_smile:.  I think **Security** makes a lot of sense; I'll move forward with making that change. Thanks!